### DISCUSSION TITLE
Install custom skills

### DISCUSSION THREAD

**Discussion by corazon008** [OpenVoiceOS/ovos-buildroot#285] [Q&A]
Hello,
I have seen somewhere that every skills is installable via `pip` but I'm trying to install [soundcloud-skill](https://github.com/OpenVoiceOS/ovos-skill-soundcloud) but I have this error. May be I can install it by cloning the git repo but I don't know how to do that
![image](https://github.com/user-attachments/assets/c0fc025f-a917-4bce-b97f-a293ed28cb4c)

**Comment by andlo** [comment]
Yes - skills are instalable by pip

https://pypi.org/project/skill-soundcloud/

The sundcloud skill should be installed by:
`pip install skill-soundcloud`

At least if you are running OVOS in a venv - i am not so sure how to install skills in a docker enviorement.
